lua/agentic/session_manager.test.lua

```
@@ -1362,4 +1362,105 @@ describe("agentic.SessionManager", function()
+                table.insert(tab_ids, tab_id)
+            end
+            for _, tab_id in ipairs(tab_ids) do
+                SessionRegistry.destroy_session(tab_id)
+            end
+        end)
+
+        it(
+            "applies default_thought_level when no model change is triggered",
+            function()
+                local tab_page_id = vim.api.nvim_get_current_tabpage()
+                local _session = SessionManager:new(tab_page_id) --[[@as agentic.SessionManager]]
```

Luacheck (1.2.0)

[warning] unused variable '_session'

(W211)